_sample_posts/2016-08-16-Measuring-software-engineering-competency.md

```diff
@@ -72,40 +74,52 @@ Now let's elaborate a little bit on the items from this list.
 
-## Do you use tasks and errors management system(s)?
+## Do you use tasks and errors management system?
 
-This, obviously, need to track what's going on on project and guarantee you will not forget about defects or tasks which should be done. Doesn’t matter what it will be - Kanban board, online tools or something else, but you have to use something to keep yourself more organised and process more transparent for any team members. 
+We need to track what's going on on the project and guarantee that defects or tasks won't be forgotten or overlooked. It doesn’t matter what it is - whiteboard with stickers on it, online tools or something else, but we have to use something to keep things organised and the whole process more transparent for everybody. 
 
 ## Do you have daily status check-in meetings?
 
-Everyone in team should know what's going on in project to avoid miscommunication issues and distractions.
+Communication is important, keep everybody in the loop but do not bore them with details. 
 
 ## Do you have short iterational process?
 
-Short process, like weeks or two, keeps team focused on a small goals  which is always good for a quality.  
+Short process, like weeks or two, keeps team focused on a small and achievable goals.
 
-## Do you have unit tests with at least 50% coverage?
+## Do you have unit tests?
 
-Unit testing is important for a code quality because it helps developers to build more flexible code design and update/refactor the code without fear to break existing functionality. It's not necessary to get crazy and always keep code coverage 100% but code should be covered by unit tests at least in half.  
+We don't know the better way to make sure that code does what it's supposed to, then to have another code that runs it and check results.
 
 ## Do you have automated end-to-end tests in DSL?
 
+Unit tests is about to keep *code quality*, but they do little help with the overall *application quality*. It doesn't matter how good is your code if application doesn't do what it supposed to do. Creating end-to-end tests in DSL achieve at least two advantages: you will be sure that application behave as expected and DSL will give you human-friendly documentation about high-level logic that cover your particular domain. It's not, actually, necessary to involve the customer or the tester to process of creating these tests, but it will be much easier to explain application behaviour and limitations to them when you have it.
```

Comment:
if an application doesn't do what it supposed to do
Creating end-to-end tests in DSL gives at least two advantages
that application behaves as expected
a human-friendly documentation about a high-level logic
to involve a customer and a tester to the process of